nautobot/extras/tests/test_api.py

```diff
@@ -2526,3 +2526,36 @@ def test_create_webhooks_with_same_content_type_same_url_common_action(self):
             response.data[0]["type_create"][0],
             "A webhook already exists for create on dcim | device type to URL http://example.com/test1",
         )
+
+    def test_patch_webhooks_with_same_content_type_same_url_common_action(self):
+        self.add_permissions("extras.change_webhook")
+
+        self.webhooks[2].payload_url = self.webhooks[1].payload_url
+        self.webhooks[2].save()
```

Comment:
Since `self.webhooks` is defined at class setup time (`setUpTestData` not `setUp`), any changes made to the instance here may propagate to other tests in this class. Make sure that's intended/desirable?

Reply:
It does not propagate to other tests in this class, I just tested to make sure